### PR TITLE
Fix prompt length calculation

### PR DESCRIPTION
**Pull request by ValekoZ**
## Description

Fix bug described in https://github.com/hugsy/gef/issues/1102

## Checklist

-  [x] My code follows the code style of this project.
-  [x] My change includes a change to the documentation, if required.
-  [x] If my change adds new code, [adequate tests](docs/testing.md) have been added.
-  [x] I have read and agree to the **CONTRIBUTING** document.

### CODE REVIEW

File: gef.py [hunk 1]
```diff
@@ -503,22 +503,22 @@ def wrapper(*args: Any, **kwargs: Any) -> Callable:
...
-        "pink"           : "\033[35m",
-        "cyan"           : "\033[36m",
-        "bold"           : "\033[1m",
-        "underline"      : "\033[4m",
-        "underline_off"  : "\033[24m",
-        "highlight"      : "\033[3m",
-        "highlight_off"  : "\033[23m",
-        "blink"          : "\033[5m",
-        "blink_off"      : "\033[25m",
+        "normal"         : "\001\033[0m\002",
```
Comment (Grazfather) [inline gef.py:506]: Let's add a comment explaining the codes (with a link).

Comment (Grazfather) [reply gef.py:506]: Also, why do we need these? Doesn't the `\033[0m` end the sequence? I think we just need to make sure to use `normal` at the end whenever we are done using a colour.

**Comment by hugsy** [comment]
@ValekoZ Can you fix the CI tests? I'll merge it afterwards

**Review by hugsy — APPROVED** [review]
LGTM

**Pull request merged**